ids/node_id.go

```diff
@@ -4,54 +4,46 @@
 package ids
 
 import (
-	"bytes"
-	"errors"
 	"fmt"
 
 	"github.com/ava-labs/avalanchego/staking"
 	"github.com/ava-labs/avalanchego/utils"
+	"github.com/ava-labs/avalanchego/utils/cb58"
 	"github.com/ava-labs/avalanchego/utils/hashing"
 )
 
-const (
-	NodeIDPrefix = "NodeID-"
-	NodeIDLen    = ShortIDLen
-)
-
 var (
-	EmptyNodeID = NodeID{}
+	EmptyNodeID = NodeID(string(EmptyShortNodeID[:]))
 
-	errShortNodeID = errors.New("insufficient NodeID length")
-
-	_ utils.Sortable[NodeID] = NodeID{}
+	_ utils.Sortable[NodeID] = (*NodeID)(nil)
 )
 
-type NodeID ShortID
-
-func (id NodeID) String() string {
-	return ShortID(id).PrefixedString(NodeIDPrefix)
-}
+type NodeID string
 
 // Any modification to Bytes will be lost since id is passed-by-value
 // Directly access NodeID[:] if you need to modify the NodeID
-func (id NodeID) Bytes() []byte {
-	return id[:]
+func (nodeID NodeID) Bytes() []byte {
```

Comment:
We can get rid of `Bytes` if this is a `string` alias now

Reply:
I'd drop it if we decide to drop Bytes() method for the other ids types.

Reply:
I think I'm fine dropping the `Bytes` method from all the IDs. Only thing we need to watch for with this is that `Bytes()` implicitly does a copy whereas `[:]` won't.